ansible_collections/arista/avd/roles/eos_designs/doc/fabric-variables.md

```diff
@@ -14,13 +14,16 @@
   - EBGP (default for l3ls-evpn)
   - IBGP (only with OSPF or ISIS variants in underlay)
   - none**
+  - her (head-end-replication)***
```

Comment:
I would keep HER upper case given that it is an acronym, in `eos_designs_facts.py` we are any way taking `lower()` for IBGP and EBGP.

Reply:
fixed